### ISSUE TITLE
The menus and menu options are a mess

### ISSUE THREAD

**Issue by merlinorg** [freezingsaddles/freezing-web#303]
Five menus with links to data pages, the organising logic eluding me.

**Comment by obscurerichard** [comment]
I agree that it's kind of a mess - and I reshuffled some things a bit earlier this year already. I moved a few items from Pointless to Data Exploration that are really more about evergreen aspects - such as Tortoise and Hare, or Points per Mile, but I was left not fully satisfied.

The fact is: we have a buttload of different ways to slice and dice this data! And that is both a joy and a sorrow. I kind of _like_ it that we're packrats about all the different ways to play with this data.  cc: @jrenaut @spfrantz

**Comment by merlinorg** [comment]
Right, I don't have an issue with the numbers of ways to slice data, it's the IA of the menus which confounds. I think possibly data exploration and pointless prize link pages might be better than menus. Was also thinking a way to star the pages you personally use, so they show as prominent links on the front page.

**Comment by obscurerichard** [comment]
#306 is a good start

**Comment by obscurerichard** [comment]
I think I have a way forward on this, see my proposal in #337

**Comment by obscurerichard** [comment]
This is not super urgent - what we have now is probably good enough for 2025. 

If we feel moved to fix this in the next 6 days, that's great, but we can also hold this one back until 2026.
